src/main/java/com/axway/apim/App.java

```diff
@@ -59,7 +59,7 @@ public static int run(String args[]) {
 					+ "  [username/password@]http://www.dneonline.com/calculator.asmx?wsdl\n"
 					+ "- a reference file called anyname-i-want.url which contains a line with the URL\n"
 					+ "  (same format as above for Swagger or WSDL).");
-				option.setRequired(true);
+				option.setRequired(false);

```

Comment:
Can you please add a comment into the usage, that -a is now optional and the apiDefinition can be provided as part of the config file?

Reply:
ok